docs/source/supported/bridges2.rst

```diff
@@ -1,4 +1,113 @@
-=============
-PSC Bridges-2
-=============
+====================
+Bridges2 (ACCESS)
+====================
 
+Platform user guide
+===================
+
+https://www.psc.edu/resources/bridges-2/user-guide-2-2/
+
+General description
+===================
+
+* Resource manager - ``SLURM``
+* Launch methods (per platform ID)
+
+  * ``access.bridges2*`` - ``MPIRUN``
+
+* Configuration per node (per platform ID)
+
+  * ``access.bridges2.RM`` (50 nodes)
+
+    * 128 CPU cores (1 thread per core)
+    * 256GB GiB of memory
+
+  * ``access.bridges2.RM-512`` (50 nodes)
+
+    * 128 CPU cores (1 thread per core)
+    * 512 GiB of memory
+
+  * ``access.bridges2.EM`` (100 nodes)
+
+    * 96 CPU cores (1 thread per core)
+    * 4 TB of memory
+
+  * ``access.bridges2.v100-32`` (24 nodes)
+
+    * 128 CPU cores (1 thread per core)
+    * 8 GPUs (Tesla v100-32) or 
+    * 512 GiB of memory
+
+  * ``access.bridges2.v100-16`` (9 node)
+
+    * 128 CPU cores (1 thread per core)
+    * 8 GPUs (Tesla v100-16)
+    * 192 GiB of memory
+
+Setup execution environment
+===========================
+
+Python virtual environment
+--------------------------
+
+**virtual environment with** ``venv`` (virtual environment with ``conda`` is
+not provided by the system)
+
+.. code-block:: bash
+
+   export PYTHONNOUSERSITE=True
+   module load python
+   python3 -m venv ve.rp
+   source ve.rp/bin/activate
```

Comment:
in configuration for Bridges2, it uses `module load anaconda3`, please check which modules are available to load python, and seems like you can provide both `venv` and `conda`

Reply:
check Polaris example - https://github.com/radical-cybertools/radical.pilot/blob/docs/nb_section3_platform_frontier/docs/source/supported/polaris.rst